frontend/app/documents/FinishOnboarding.tsx

```
@@ -0,0 +1,267 @@
+  const updateContractor = trpc.companyInviteLinks.completeOnboarding.useMutation({
+    onSuccess: async () => {
+      await trpcUtils.documents.list.invalidate();
+      onNext();
+    },
+  });
+  const submit = form.handleSubmit((values) => {
+    updateContractor.mutate({ companyId: company.id, ...values, startedAt: values.startedAt.toString() });
+  });
+
+  return (
+    <Dialog open={open} onOpenChange={() => {}}>
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)